public/plugin.ts

```diff
@@ -66,6 +66,16 @@ async function hasApiPermission(core: CoreSetup): Promise<boolean | undefined> {
 const DEFAULT_READONLY_ROLES = ['kibana_read_only'];
 const APP_ID_HOME = 'home';
 const APP_ID_DASHBOARDS = 'dashboards';
+// Kibana app is for legacy url migration
+const APP_ID_KIBANA = 'kibana';
+const APP_ID_SHORT_URL_REDIRECT = '"short_url_redirect"';
```

Comment:
would you elaborate a little why `short_url_redirect` as AppId is enclosured with double quote?

Reply:
That was a mistake. Fixing.

Reply:
Removed as it is handled by server.